target/linux/mvebu/cortexa53/base-files/etc/board.d/02_network

```diff
@@ -21,9 +21,16 @@ globalscale,espressobin-ultra)
 	ucidef_set_interfaces_lan_wan "lan0 lan1 lan2 lan3" "wan"
 	;;
 marvell,armada-3720-db|\
-methode,udpu|\
+methode,udpu)
+	ucidef_set_interfaces_lan_wan "eth1" "eth0"
+	;;
 methode,edpu)
+	# eDPU+ has a 88E6361 switch, so we check for it
+	if ip link | grep -q uplink; then
+	ucidef_set_interfaces_lan_wan "downlink" "uplink"
```

Comment:
Is this a way to differentiate them? Why do we need to use different lables?

Reply:
The only way to differentiate them from OpenWrt is by the `uplink` and `downlink` interfaces.

We need different names as that is what is printed on the device case and customers prefer for it to match 1:1.

Reply:
> We need different names as that is what is printed on the device

oh ok!